Formula/q/qbittorrent-cli.rb

```diff
@@ -0,0 +1,73 @@
+class QbittorrentCli < Formula
+  desc "Command-line interface for qBittorrent written in Go"
+  homepage "https://github.com/ludviglundgren/qbittorrent-cli"
+  url "https://github.com/ludviglundgren/qbittorrent-cli/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "91969f22bb167f99091a1ff8f4dbfe61cb1b592cb000453859ca9173ecbb8f10"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "cmd/qbt/main.go"
+  end
+
+  # helper function for HTTP communications test
+  def send_response(client, payload)
+    client.puts "HTTP/1.1 200 OK"
+    client.puts "content-type: text/plain; charset=UTF-8"
+    client.puts "content-length: #{payload.length}"
+    client.puts ""
+    client.write payload.to_s
+    client.flush
+  end
+
+  test do
+    # test if qbittorent-cli version works
+    # (output is currently not very helpful though)
+    expected_output = <<~EOS
+      qbt - qbitttorrent cli
+      Version: dev
+      Commit: none
+      Date: unknown
```

Comment:
the version seems a bit off, maybe we can override the ldflags?

Reply:
Ok, I have learned now how to do this :sweat_smile: 

Do you have a suggestion on how to retrieve the commit associated to a release/tag in a Homebrew context? I have implemented a way that works in principle but will run into Github API limits. It would be also possible to just set the commit to "unknown" or similar as an alternative.